Remotion/Validation/Globalization.UnitTests/LocalizedValidationMessageFactoryTest.cs

```diff
@@ -43,7 +43,7 @@ public void CreateValidationMessageForPropertyValidator_WithEqualValidator_Retur
       Assert.That(validationMessage, Is.Not.Null);
       Assert.That(validationMessage, Is.InstanceOf<ResourceManagerBasedValidationMessage>());
 
-      Assert.That(validationMessage.ToString(), Is.EqualTo("Enter the value \"{0}\"."));
+      Assert.That(validationMessage.ToString(), Is.EqualTo("Enter the value {0}."));
```

Comment:
we keep the quotation marks for this one here. the semantics are slightly different.

And yes, this means we have one test that is different in the localization integration test. there, I would suggest we make a regex that tests qutation marks: `"4"|«4»`.